src/main/java/org/edumips64/webclient.gwt.xml

```diff
@@ -2,6 +2,14 @@
   <inherits name="com.google.gwt.user.User"/>
   <inherits name="com.google.gwt.logging.Logging"/>
   <inherits name="com.google.gwt.json.JSON" />
+
+	<inherits name="org.gwtproject.rpc.worker.RpcToWorkers" />
+
+  <inherits name="prj.gwtwwlinker.workerlinker.WorkerLinker" />
+  <!-- Settings required for Web Worker support. A single permutation must be compiled. -->
+  <set-property name="user.agent" value="safari" />
+  <set-configuration-property name="user.agent.runtimeWarning" value="false" />
```

Comment:
this is the right idea to disable the checks, but this is the user agent check, not the compatMode check, you want both turned off. If you are able to simplify your .gwt.xml to remove the other modules though, these are unnecessary (and will cause errors if they are present).

Reply:
Yes, thanks a lot! It seems to compile correctly now. Still not working fully but I have something to work with now!